containerd/metadata.csv

```diff
@@ -0,0 +1,39 @@
+containerd.hugetlb.usage,gauge,,bytes,,The hugetlb usage,0,containerd, hugetlb usage
+containerd.mem.current.usage,gauge,,bytes,,The memory usage,0,containerd,mem_usage
+containerd.mem.current.failcnt,gauge,,,,The usage failcnt,0,containerd,mem_usage_failcnt_total
+containerd.mem.current.limit,gauge,,bytes,,The memory limit,0,containerd,mem_usage_limit
+containerd.mem.current.max,gauge,,bytes,,The memory maximum usage,0,containerd,mem_usage_max
+containerd.mem.kernel_tcp.usage,gauge,,bytes,,The kerneltcp usage,0,containerd, mem_kerneltcp_usage
+containerd.mem.kernel_tcp.failcnt,gauge,,bytes,,The kerneltcp failcnt,0,containerd, mem_kerneltcp_failcnt_total
+containerd.mem.kernel_tcp.limit,gauge,,bytes,,The kerneltcp limit,0,containerd, mem_kerneltcp_limit
+containerd.mem.kernel_tcp.max,gauge,,bytes,,The kerneltcp maximum usage,0,containerd, mem_kerneltcp_max
+containerd.mem.kernel.usage,gauge,,bytes,,The kernel usage,0,containerd, mem_kernel_usage
+containerd.mem.kernel.failcnt,gauge,,bytes,,The kernel failcnt,0,containerd, mem_kernel_failcnt_total
+containerd.mem.kernel.limit,gauge,,bytes,,The kernel limit,0,containerd, mem_kernel_limit
+containerd.mem.kernel.max,gauge,,bytes,,The kernel maximum usage,0,containerd, mem_kernel_max
+containerd.mem.swap.usage,gauge,,bytes,,The swap usage,0,containerd,mem_swap_usage
+containerd.mem.swap.failcnt,gauge,,bytes,,The swap failcnt,0,containerd, mem_swap_failcnt_total
+containerd.mem.swap.limit,gauge,,bytes,,The swap limit,0,containerd,mem_swap_limit
+containerd.mem.swap.max,gauge,,bytes,,The swap maximum usage,0,containerd,mem_swap_max
+containerd.mem.cache,gauge,,bytes,,The cache amount used,0,containerd, cache amount
+containerd.mem.rss,gauge,,bytes,,The rss amount used,0,containerd, mem_rss
+containerd.mem.rsshuge,gauge,,bytes,,The rss_huge amount used,0,containerd, mem_rss_huge
+containerd.mem.dirty,gauge,,bytes,,The dirty amount,0,containerd, mem dirty
+containerd.cpu.throttled.periods,rate,,nanoseconds,,The total cpu throttled time,0,containerd,cpu throttled time
+containerd.cpu.system,rate,,nanoseconds,,The total cpu time,0,containerd,cpu total
+containerd.cpu.total,rate,,nanoseconds,,The total cpu time,0,containerd,cpu total
+containerd.cpu.user,rate,,nanoseconds,,The total user cpu time,0,containerd,cpu user
+containerd.cpu_total,gauge,,nanoseconds,,The total cpu time,0,containerd,cpu total
+containerd.cpu_user,gauge,,nanoseconds,,The total user cpu time,0,containerd,cpu user
+containerd.blkio.merged_recursive,rate,,bytes,,The blkio io merged recursive,0,containerd,io merged recursive
+containerd.blkio.queued_recursive,rate,,bytes,,The blkio io queued recursive,0,containerd,io queued recursive
+containerd.blkio.sectors_recursive,rate,,bytes,,The blkio io sectors recursive,0,containerd,io sectors recursive
+containerd.blkio.service_recursive_bytes,rate,,bytes,,The blkio io service bytes recursive,0,containerd, blkio service bytes recursive
```

Comment:
a bunch of these metrics are not in `byte` see https://github.com/containerd/containerd/blob/master/metrics/cgroups/blkio.go